inductiva/templates/wind_terrain/openfoam/files/0/include/sideAndTopPatches

```diff
@@ -0,0 +1,24 @@
+/*--------------------------------*- C++ -*----------------------------------*\
+  =========                 |
+  \\      /  F ield         | OpenFOAM: The Open Source CFD Toolbox
+   \\    /   O peration     | Website:  https://openfoam.org
+    \\  /    A nd           | Version:  8
+     \\/     M anipulation  |
+\*---------------------------------------------------------------------------*/
+
+top
+{
+    type slip;
+}
```

Comment:
Do you know what type of boundary conditions this refers to?

Reply:
https://www.cfdsupport.com/openfoam-training-by-cfd-support/node118.html